.env.example

```
@@ -48,6 +48,14 @@ FLUENT_AI_API_PREFIX=
 # Key used to authenticate incoming webhook callbacks from fluent-ai
 AI_INBOUND_SERVICE_KEY=generate-a-32-character-secret-key
 
+# Aquifer (translation resources: TN / TQ / Images & Maps)
+# Base URL of the Aquifer API (no trailing slash). Defaults to production if unset.
+AQUIFER_API_URL=https://api.aquifer.bible
+# Server-held Aquifer API key — do not ship this to mobile/web clients.
+# Optional: leave blank to boot without Aquifer; TN/TQ/Images/manifest return 502.
+# Request a key at https://www.aquifer.bible/apiaccess
+AQUIFER_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AQUIFER_API_KEY key should go before the AQUIFER_API_URL key

(UnorderedKey)